firmware/src/error_handling_flow/error_handling_flow.cpp

```diff
@@ -15,16 +35,34 @@ void ErrorHandlingFlow::handleEvent(WiFiEvent event)
     motor_notifier->requestUpdate(blocked_motor_config);
     switch (event.type)
     {
-    case SK_MQTT_CONNECTION_FAILED:
     case SK_MQTT_RETRY_LIMIT_REACHED:
+        if (!WiFi.isConnected())
+        {
+            sprintf(ip_data, "%s", "http://192.168.4.1/mqtt"); // always the same
+        }
+        else
+        {
+            sprintf(ip_data, "http://%s/mqtt", WiFi.localIP().toString().c_str());
```

Comment:
In theory, we should not access global objects like that. But we don't have a good mechanism to pass this data around yet.

Reply:
WiFi.localIP()?